source/community/policy/editorial.md

```diff
@@ -92,27 +92,29 @@ In the event that an editor disagrees with a merge that has already happened, th
 ### 2.4 Face to Face Meetings
 {: #face-to-face-meetings}
 
-Editors will meet at least four times per year, and all editors _MUST_ attend at least three of those meetings per calendar year. Every effort will be made to co-locate at least one of the meetings with another meeting at which most of editors are already likely to be present.
+Editors will meet at least three times per calendar year, and all editors _MUST_ attend at least two of those meetings. Every effort will be made to co-locate at least one of the meetings with another meeting at which most of editors are already likely to be present, such as the IIIF Working Session and IIIF Conference. One of these meetings _MAY_ be conducted virtually as a multi-day teleconference.
```

Comment:
True ... I wanted to avoid the implication that we have a meeting every other week that would count towards the "face to face" meetings. Suggestions for wording?

Reply:
...as an extended teleconference ?